python/ray/autoscaler/_private/node_launcher.py

```diff
@@ -76,9 +74,6 @@ def _launch_node(self, config: Dict[str, Any], count: int,
             # second create time 4 times.
             self.prom_metrics.worker_create_node_time.observe(launch_time)
         self.prom_metrics.started_nodes.inc(count)
-        after = self.provider.non_terminated_nodes(tag_filters=worker_filter)
-        if set(after).issubset(before):
-            self.log("No new nodes reported after node creation.")
 
```

Comment:
is this not important?

Reply:
Checking the list of nodes immediately after creating is no longer allowed -- you have to wait until the next update.

It's possible to do this check at the beginning of the next update, but it's not worth it right now.